content/docs/addons-shallow-renderer.md

```diff
@@ -6,31 +6,31 @@ layout: docs
 category: Reference
 ---
 
-**Importing**
+**Cara Import**
 
 ```javascript
 import ShallowRenderer from 'react-test-renderer/shallow'; // ES6
 var ShallowRenderer = require('react-test-renderer/shallow'); // ES5 with npm
```

Comment:
Whoops, looks like i missed this, please translate comments and strings inside codeblock 😄

Reply:
alright 😄